halyard-config/src/main/java/com/netflix/spinnaker/halyard/config/validate/v1/canary/google/GoogleCanaryAccountValidator.java

```diff
@@ -30,16 +30,13 @@
 import lombok.Data;
 import lombok.EqualsAndHashCode;
 import lombok.Setter;
-import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.scheduling.TaskScheduler;
-import org.springframework.stereotype.Component;
 
 @Data
 @EqualsAndHashCode(callSuper = false)
-@Component
 public class GoogleCanaryAccountValidator extends CanaryAccountValidator {
 
-  @Autowired
+  @Setter
```

Comment:
Do you need an `@Setter` if the class has `@Data`?

Reply:
Nope, fixed.